Pipfile.lock

```diff
@@ -1,557 +0,0 @@
-{
```

Comment:
I don't think you want to delete all this? At least there were no corresponding changes to the `Pipfile` that went with this to lead to a delete like this.

Reply:
forgot to run pipenv lock!